.eslintrc

```diff
@@ -14,7 +14,7 @@
   },
   "overrides": [
     {
-      "files": [ "stories.js", "test.js", "src/stories/index.js" ],
+      "files": [ "stories.js", "test.js", "src/stories/index.js", "src/setupTests.js" ],
```

Comment:
perhaps we should have a src/tests folder and wildcard that, along with wildcarding src/stories rather than specific files, but that doesn't need to be addressed now

Reply:
agreed